doc/develop/application/index.rst

```diff
@@ -1011,10 +1011,15 @@ for additional information on how to flash your board.
 Running in an Emulator
 ======================
 
-The kernel has built-in emulator support for QEMU (on Linux/macOS only, this
-is not yet supported on Windows). It allows you to run and test an application
-virtually, before (or in lieu of) loading and running it on actual target
-hardware. Follow these instructions to run an application via QEMU:
+The kernel has built-in emulator support for QEMU (on Linux/macOS only).
+It allows you to run and test an application virtually, before
+(or in lieu of) loading and running it on actual target hardware.
+
+On Windows, you need to install QEMU manually from
+`Download QEMU <https://www.qemu.org/download/#windows>`_. After installation,
+add path to QEMU installation folder to PATH environment variable.
```

Comment:
Not sure "the kernel" is used correctly here. I'd rephrase it to say that "Zephyr has ..." and remove the "(on Linux/macOS only)".

Also, I think mentioning how to install QEMU in the beyond-GSG is sufficient. No need to repeat it here.